attestation-agent/kbs_protocol/src/bin/trustee-attester/README.md

```diff
@@ -0,0 +1,33 @@
+# Trustee attester #
```

Comment:
The preview looks good, while we do not need the `#` in the end of the line when making headings. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings